proposals/0240-sdl-js-pwa.md

```diff
@@ -257,7 +272,7 @@ Many services are available over a web application and modern WebEngines provide
   <param name="nicknames" type="String" minlength="0" maxlength="100" array="true" minsize="0" maxsize="100" mandatory="false">
       <description>An array of app names an app is allowed to register with. If included in a SetAppProperties request, this value will overwrite the existing "nicknames" field in the app policies section of the policy table.</description>
   </param>
-  <param name="appID" type="String" maxlength="100" mandatory="true"/>
+  <param name="policyAppID" type="String" maxlength="100" minlength="1" mandatory="true" />
```

Comment:
@kshala-ford Shouldn't this parameter be renamed in `GetAppProperties` request ?

Reply:
@AByzhynar please see item 7 in this comment for the agreed to param name: https://github.com/smartdevicelink/sdl_evolution/issues/767#issuecomment-562608883

Reply:
> @AByzhynar please see item 7 in this comment for the agreed to param name: [#767 (comment)](https://github.com/smartdevicelink/sdl_evolution/issues/767#issuecomment-562608883)

@theresalech You are absolutely right. But parameter appID was present in both `<function name="GetAppProperties" messagetype="request">` and `<function name="GetAppProperties" messagetype="response">` in `AppProperties` structure. 

Please see the following : 
```cpp
<function name="GetAppProperties" messagetype="request">
  <description>
    HMI >SDL. RPC used to get the current properties of an application
  </description>
  <param name="appID" type="String" maxlength="100" mandatory="false">
    If specified the response will contain the properties of the specified app ID.
    Otherwise if omitted all app properties will be returned at once.
  </param>
</function>

<function name="GetAppProperties" messagetype="response">
  <description>The response to GetAppProperties</description>
  <param name="properties" type="AppProperties" array="true" minsize="1" mandatory="false">
    <description>The requested application properties</description>
  </param>
 
</function>

<struct name="AppProperties">
  <param name="nicknames" type="String" minlength="0" maxlength="100" array="true" minsize="0" maxsize="100" mandatory="false">
      <description>An array of app names an app is allowed to register with. If included in a SetAppProperties request, 
                   this value will overwrite the existing "nicknames" field in the app policies section of the policy table.
      </description>
  </param>
  <param name="policyAppID" type="String" maxlength="100" minlength="1" mandatory="true" />
  ...
</struct>
```

As I understand parameter was renamed only in response which leads to mismatch between `GetAppProperties` request and `GetAppProperties` response.

Reply:
The voted upon proposal did not include these changes., therefore if you would like to see those changes they will need to be part of revision. Changes to the proposal can't be done outside the steering committee and as such this is not the forum to discuss nor accept these changes.